src/api/models/ActionModel.js

```diff
@@ -16,12 +16,14 @@ const ActionSchema = new mongo.Schema({
     },
   },
   date: { type: Date, required: true },
+  dateCreated: { type: Date },
```

Comment:
Can you set the date as a default value similar to how it is for `approved`?
(The same applies for TeamModel)